enterprise/frontend/src/embedding-sdk/test/CommonSdkStoryWrapper.tsx

```diff
@@ -11,7 +11,10 @@ const METABASE_JWT_SHARED_SECRET =
 
 const secret = new TextEncoder().encode(METABASE_JWT_SHARED_SECRET);
 
-const DEFAULT_CONFIG: SDKConfig = {
+/**
+ * SDK config that signs the jwt on the FE
+ */
+export const storybookSdkDefaultConfig: SDKConfig = {
```

Comment:
exporting this as I need to use directly (without the decorator) in some stories, as I need to change some settings